### ISSUE TITLE
How to not interrupt previous job if it is time for the job to start again?

### ISSUE THREAD

**Issue by jasonxeno** [percolatestudio/meteor-synced-cron#40]
I have a job that I want to run everyone 1 minute. But sometimes it takes more than one minute to complete the job, when this happens, that job is terminated in the middle of it running and starts a new job.

Is there a way to let the prevous job finish before starting over again,  if it is still running? Or treat the new job as a new instance that does not stop the prevous running job if the new job is started before the old one is done?

Basically it is dangerous in my app to interrupt the job so I want a way to prevent the job being stopped by the next scheduled run when the job getting ran again.

**Comment by zol** [comment]
Hey @jasonxeno . There is no code in synced-cron to terminate jobs. What you're outlining should work, it sounds like there may be a bug in your job code.

**Comment by tracethepath** [comment]
I have a synchedcron job which calls an API and inserts data in a collection. I dont want to start the job if its already existing or was triggered by an other session. Pls let me know the code to do the same.
